conan/cli/commands/graph.py

```diff
@@ -234,7 +239,10 @@ def graph_info(conan_api, parser, subparser, *args):
     return {"graph": deps_graph,
             "field_filter": args.filter,
             "package_filter": args.package_filter,
-            "conan_api": conan_api}
+            "conan_api": conan_api,
+            "conan_error": deps_graph.error,
+            # Do not compute graph errors if there are dependency errors
+            "conan_warning": InstallGraph(deps_graph).get_errors() if not deps_graph.error else None}
```

Comment:
This ``conan_warning: ...get_error()`` is a bit weird, it should probably be converted to string at this point